searx/static/themes/simple/xsl/rss.xsl

```diff
@@ -1,59 +1,39 @@
 <?xml version="1.0"?>
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-
   <xsl:output method="html" version="5" encoding="UTF-8" indent="yes" />
-
   <xsl:template match="rss">
-	<html xmlns="http://www.w3.org/1999/xhtml">
-        <head>
-            <title><xsl:value-of select="channel/title" /> RSS Feed</title>
-            <meta charset="UTF-8" />
-            <meta http-equiv="x-ua-compatible" content="IE=edge,chrome=1" />
-            <meta name="viewport" content="width=device-width,initial-scale=1" />
-            <style type="text/css">
-				html {
-					background: #fff;
-					color: #444;
-				}
-			
-                a {
-					color: #334999;
-					text-decoration: none;
-				}
-
-				a:hover {
-					text-decoration: underline;
-				}
-            </style>
```

Comment:
I think there is no need to style the generated HTML .. the default rendering for HTML elements is good enough.

Reply:
I personally prefer styling this a bit as I hate the default `<a>` styling of HTML, and that small CSS part here is not much of an overhead (just 10 lines).

Reply:
OK, but then CSS should be implemented in our LESS workflow.

I pushed a commit ``[mod] add CSS (LESS) to xsl style to view rss in browser`` on top of your branch .. may you might have a look at / thanks :+1: